hardware/raspberrypi/booteeprom.md

```diff
@@ -44,6 +45,8 @@ sudo systemctl unmask rpi-eeprom-update
 
 The `FREEZE_VERSION` option in the EEPROM config file may be used to indicate that the EEPROM should not be updated on this board. 
 
+Note than updating the bootloader (automatic or manual) will retain any custom configuration options of the previous installed version.
```

Comment:
I think if you do a "manual update" and use the `-d` option, then it _doesn't_ retain any custom configuration options? @timg236 ?

Reply:
Correct "357    -d Use the default bootloader config instead of migrating the current settings" 
where default is the embedded config in that image